extensions/wikia/AdEngine/js/lookup/rubicon/rubiconFastlane.js

```diff
@@ -176,20 +177,26 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane', [
 
 	function saveBestPrice(slotName, tiers) {
 		tiers.forEach(function (tier) {
-			bestPrices[slotName] = Math.max(rubiconTier.parsePrice(tier), bestPrices[slotName] || 0);
+			bestPrices[slotName] = Math.max(rubiconTier.parseOpenMarketPrice(tier), bestPrices[slotName] || 0);
+			bestPricesPrivate[slotName] = Math.max(rubiconTier.parsePrivatePrice(tier), bestPricesPrivate[slotName] || 0);
 		});
 	}
 
 	function getBestSlotPrice(slotName) {
+		return {
+			fastlane: getBestPriceString(bestPrices[slotName]),
+			fastlane_private: getBestPriceString(bestPricesPrivate[slotName])
+		};
+	}
+
+	function getBestPriceString(bestPriceForSlot) {
 		var price;
 
-		if (typeof bestPrices[slotName] !== 'undefined') {
-			price = (bestPrices[slotName] / 100).toFixed(2).toString();
+		if (typeof bestPriceForSlot !== 'undefined') {
```

Comment:
in case of `0` the `(bestPrices[slotName] / 100).toFixed(2).toString();` wont be performed which is bad